usmqe/web/skyring/loginpage/pages.py

```diff
@@ -0,0 +1,71 @@
+"""
+Login page abstraction.
+"""
+
+
+import pytest
+
+from webstr.core import WebstrPage
+from webstr.selenium.ui.support import WaitForWebstrPage
+
+from usmqe.web.skyring.mainpage.navpage import pages as navpage
+from usmqe.web.skyring.loginpage import models as m_loginpage
+
+
+class LoginPage(WebstrPage):
+    """
+    Base page object for login page.
+
+    Parameters:
+        _model - page model
+        _label - a read-able description of a class
+        _required_elems - web elements to be checked
+    """
+# not working during import the usm_web_url does not exist
+#    _location = pytest.config.getini("usm_web_url")
+    _model = m_loginpage.LoginPageModel
+    _label = 'login page'
+    _required_elems = ['username', 'password', 'login_btn']
+
+    def __init__(self, driver):
+        """ init
+        Note: _location has to be initialized here as it needs
+            a config parameter which is not know before the actual run
+            _location - initial URL to load upon instance creation
+
+        Atributes:
+            driver: web driver
+        """
+        self._location = pytest.config.getini("usm_web_url")
```

Comment:
Would it be possible to pass the location from the test case? That way we would not have to import pytest from pages and it woud give us more flexibility (eg. when we would like to do some test cases with a modified url to test redirection or some negative cases).